test/test_utils/ColdResumeManager.cpp

```diff
@@ -1,6 +1,6 @@
 // Copyright 2004-present Facebook. All Rights Reserved.
 
-#include "test/test_utils/ColdResumeManager.h"
```

Comment:
We had this issue for one more header in fbcode. I wonder what's the real cause of requiring this update?